src/modules/DhcpServer.pm

```diff
@@ -1514,11 +1514,11 @@ DHCP server will not be available."));
     @allowed_interfaces = split (/ /, $ifaces_list);
 
     # if firewall is enabled
-    if (SuSEFirewall->GetEnableService()) {
+    if (FirewalldWrapper->is_enabled()) {
 	foreach my $iface (@allowed_interfaces) {
-	    my $iface_zone = SuSEFirewall->GetZoneOfInterface($iface);
+	    my $iface_zone = FirewalldWrapper->zone_name_of_interface($iface);
```

Comment:
Has you confirmed that the default_zone is not used when no zone is returned? If the behavior is the same, then It is fine.

Reply:
Checked, seems it is the same behavior so LGTM

Reply:
As far I see....yes